requirements/freeze.py

```diff
@@ -18,7 +18,7 @@
 
 parser = argparse.ArgumentParser(description=__doc__.strip())
 parser.add_argument("--venv", default=None)
-parser.add_argument("--keep-venv", action="store_true")
+parser.add_argument("--keep-venv", action="store_true", dest='keep')
```

Comment:
L64 passes this as `args.keep`, opting to set that name here via `dest` instead of changing L64 to use `keep_venv`.